var/spack/repos/builtin/packages/py-openai-gym/package.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+class PyOpenaiGym(PythonPackage):
```

Comment:
```suggestion
class PyGym(PythonPackage):
```
We usually use the PyPI name when deciding the package name for Python packages